src/lib/adwaitacolors.cpp

```diff
@@ -112,7 +112,8 @@ QColor Colors::lighten(const QColor &color, qreal amount)
 QColor Colors::mix(const QColor &c1, const QColor &c2, qreal bias)
 {
     auto mixQreal = [](qreal a, qreal b, qreal bias) {
-        return a + (b - a) * bias;
+        /* libsass-compilant blending */
+        return bias * b + (1 - bias) * a;
```

Comment:
Sorry, I wrote this a week ago at least, but it was not sent until I did the review.

Reply:
It is directly related to unfocused views fix because in `src/style/adwaitastyle.cpp` inactive color is calculated using `Colors::mix` so it has to use the same algo as in `libsass` to generate exactly the same color and be able to compare it  with a color from `Adwaita-*.scss`: 

```
$osd_insensitive_fg_color: mix($osd_fg_color, opacify($osd_bg_color, 1), 50%);
```

Reply:
If you wish I can make it a separate commit, anyway I think it's a correct change because the Adwaita palette uses this algorithm (since colors are defined in `_colors.scss`). Previous version of `Colors::mix` was simply taken from KDE frameworks, which did not assume compatibility with Adwaita